src/_scss/pages/agencyV2/statusOfFunds/_introSection.scss

```diff
@@ -1,3 +1,5 @@
+@import 'core/variables';
```

Comment:
The variables should be available without needing to import in each file. (They are imported by `all.scss` which is imported at the page level)